ramp-database/ramp_database/tools/leaderboard.py

```diff
@@ -461,12 +481,23 @@ def get_leaderboard(
             for sub in submissions
         ]
         df = pd.DataFrame(data, columns=columns)
+        if leaderboard_type == "new":
+
+            def create_button(cell_value):
+                return f"<button onclick=\"alert(('You killed ' \
+                    + 'submission {cell_value}!'))\">{cell_value}</button>"
+
+            df["Kill Sub"] = df["submission_id"].apply(
+                lambda x: create_button(x)
+            )
     else:
         # make some extra filtering
         submissions = [sub for sub in submissions if sub.is_public_leaderboard]
         if not submissions:
             return None
-        competition_type = "public" if "public" in leaderboard_type else "private"
+        competition_type = (
+            "public" if "public" in leaderboard_type else "private"
+        )
```

Comment:
I think if `leaderboard_type not in ['new', 'failed']` we need to delete the extra column "id" we created before `_compute_competition_leaderboard` is run otherwise it will show in all other leaderboards

Reply:
Since we create this extra column in ```elif leaderboard_type in ["new", "failed"]:``` it seems that we don't need to delete it in other leaderboard types